c/driver/postgresql/postgresql_test.cc

```diff
@@ -539,40 +539,6 @@ TEST_F(PostgresConnectionTest, GetObjectsTableTypesFilter) {
   ASSERT_NE(view, nullptr) << "did not find view adbc_table_types_view_test";
 }
 
-TEST_F(PostgresConnectionTest, MetadataGetTableSchemaInjection) {
```

Comment:
We don't necessarily need to remove this test, but it validates the same behavior as what was added to the adbc_validation.h here